src/index.jsx

```diff
@@ -1,6 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import MainRoute from "./routes/MainRoute";
+import MenuContainer from "./components/MenuContainer";
+
+ReactDOM.render(
+    <MenuContainer />,
+    document.querySelector("#contentContainer")
+);
```

Comment:
Just a heads up - is there any particular reason we need to affix multiple React components directly to the HTML file? Typically, the idea of the Route model is to organize React components within the route (MainRoute.tsx in our case), such that any interaction between those separate components can be facilitated by the MainRoute. Having them all served through the MainRoute also allows for us to keep the data shared across the page in the route file.
 
Could we have the MenuContainer and any HTML content moved into the MainRoute? Thanks, and sorry for the trouble!

- [ ] Move MenuContainer and HTML content into MainRoute